src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -83,7 +83,8 @@
   <Target Name="CreateTestListJson"
           DependsOnTargets="CreateFuncTestListJson;CreatePerfTestListJson" />
 
-  <Target Name="CreateFuncTestListJson" DependsOnTargets="CreateAzureStorage">
+  <!-- allow skipping this target if perf testing is enabled -->
+  <Target Name="CreateFuncTestListJson" DependsOnTargets="CreateAzureStorage" Condition="'$(Performance)' != 'true' or '$(FuncTestsDisabled)' != 'true'">
```

Comment:
This line is a bit long.

I'd also prefer it wasn't a negative property. Instead of "disablefoo" that defaults to false, I'd rather have it be "enablefoo" that defaults to true... "DisableFoo=false" to not not do a thing always hurts my brain, and I frequently get it backward.